src/audio/mic_privacy_manager/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+add_local_sources(sof mic_privacy_manager_intel.c)
```

Comment:
This isn't used by Zephyr builds, right? Because you also add this to zephyr/CMakeLists.txt. So it looks like you also need this file for XTOS builds? But it uses calls like `LOG_DBG()` which are Zephyr-specific?

Reply:
This CMake file will be added only if Mic privacy kconfig is set. I think the future plan is to get rid of giant zephyr cmake file and mic privacy cmake is already ready for such move

Reply:
@abonislawski ok, yes, correct, we plan to move back to a proper cmake tree-based structure, and it's good that you're already thinking about it! It's just that (1) we don't know exactly yet what it will look like, so it might happen that changes will be needed to this file then, and (2) it's a bit confusing now to add unused files. So maybe at least add a comment there along the lines of "currently unused, but will be used in the future when src/audiuo files are removed from zephyr/CMakeLists.txt"